browser/js/identities.js

```diff
@@ -53,47 +53,47 @@ const identityMatchers = [
   {
     service: "keybase",
     getUsername: function(loc) { return loc.pathname.split('/')[1]; },
-    locationMatches: new RegExp('\\.keybase\\.io/([\\w]+)[/]?'),
-    originAndPathMatches: 'keybase\\.io/[\\w]+[/]?',
+    pathMatches: new RegExp('^([\\w]+)[/]?'),
+    originAndPathMatches: '^https://keybase\\.io/[\\w]+[/]?',
     hostEquals: ['keybase.io'],
     css: ['.profile-heading']
   },
   {
     service: "reddit",
     getUsername: function(loc) { return loc.pathname.split('/')[2]; },
-    locationMatches: new RegExp('\\.reddit\\.com/user/([\\w-]+)[/]?$'),
-    originAndPathMatches: '\\.reddit\\.com/user/[\\w-]+[/]?$',
+    pathMatches: new RegExp('^/user/([\\w-]+)[/]?$'),
+    originAndPathMatches: '^https://(www)?\\.reddit\\.com/user/[\\w-]+[/]?$',
```

Comment:
There's a whole slew of reddit subdomains, like `np.` etc. Might want to find a full list.

Reply:
kk